cmd/px-node-wiper/px-node-wiper.sh

```diff
@@ -171,14 +157,13 @@ if [ "$REMOVE_DATA" = "1" ]; then
     touch "$STATUS_FILE"
     if [ "$WAIT" = "1" ]; then
       echo "Successfully wiped px. Sleeping..."
-      sleep_forever
+      sleep 1800
     fi
   else
       fatal "error: remove pwx configuration failed with code: $?"
   fi
 else
   touch "$STATUS_FILE"
   echo "Successfully wiped px. Sleeping..."
-  sleep_forever
+  sleep 1800
```

Comment:
Putting `sleep 60` into the background, and then waiting for it -- and all in a loop is an unnecessary complication.  We get the same effect with a "very long sleep".

The _"sleep at the end"_ is required because without it, the POD's executable will terminate and K8s will restart the POD.   Granted, if this were to happen, re-running the node-wiper POD will not have any bad effect, since the data/files are already removed.